### PR TITLE
Additional template variables for vaccines manuscripts

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

There were a few notes in the vaccines manuscripts to add template variables that were not generated. 

This PR adds:
- owid_total_countries: a count of the total number of countries with vaccine data in OWID's dataset
- viper_date_pretty: extracts the date of last update from the HTML on the viper site
- Sinovac_country_count: the number of countries for which Sinovac Coronavac distribution data is available
- Sinovac_continents: a string indicating the names of the continents where Sinovac Coronavac is distributed (formatted as "X, Y, and Z")
- Covaxin_country_count: the number of countries for which Covaxin distribution data is available
- Covaxin_continents: a string indicating the names of the continents where Covaxin is distributed (formatted as "X, Y, and Z")

It also adds a period to the end of the sentences generated saying no approved vaccines in the dataset were developed on a continent

Additionally, all variables in the JSON are now strings (probably doesn't matter, but I hadn't been careful about this, and it seems strategic for the sake of future sanity...)

### Related issues

#1167 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: owiddata/01.OWID.basicStats.py [hunk 1]
```diff
@@ -21,6 +22,7 @@ def convert_date(git_date):
     return datetime.datetime.fromisoformat(git_date).strftime('%B %d, %Y').replace(' 0', ' ')
 
 def billions(count, decimals=0):
+    """Cleans integer in the billions to make a """
```
Comment (agitter) [inline owiddata/01.OWID.basicStats.py:25]: Is part of the comment missing?

File: owiddata/01.OWID.basicStats.py [hunk 2]
```diff
@@ -70,13 +71,29 @@ def main(args):
...
+        owid_stats[vaccine+"_country_count"] = str(len(isos))
+
+        country_data = setup_geopandas()
+        continents_df = (
+                country_data[country_data['iso_a3'].isin(isos)]
+                >> ply.select("continent")
+                >> ply.distinct(["continent"])
+        )
+        continents = [value for row in continents_df.values.tolist() for value in row]
+        owid_stats[vaccine + "_continents"] = ', '.join(continents[:-1]) + " and " + continents[-1]
```
Comment (agitter) [inline owiddata/01.OWID.basicStats.py:95]:
```suggestion
        owid_stats[vaccine + "_continents"] = ', '.join(continents[:-1]) + ", and " + continents[-1]
```
Should we sort the list of continents? It seems fairly arbitrary now.

**Review by agitter — COMMENTED** [review]
This generally looks good.  I have a couple small comments.

As long as you are doing minor formatting cleanup, what do you think about changing the y-axis formatting of `OWID_doseTypebyGDP.png` to be in the units of millions?

File: owiddata/01.OWID.basicStats.py (hunk 1 above)
Comment (rando2) [reply owiddata/01.OWID.basicStats.py:25]:
```suggestion
    """Cleans integer in the billions to make a string formatted as 'X billion', defaulting to integer"""
```

Comment (rando2) [reply owiddata/01.OWID.basicStats.py:25]: Yes! Weird, I thought I typed multiple lines, but clearly something went wrong.

File: owiddata/01.OWID.basicStats.py (hunk 2 above)
Comment (rando2) [reply owiddata/01.OWID.basicStats.py:95]: I'm also adding the Oxford comma!

**Review by rando2 — COMMENTED** [review]
Thank you @agitter! I think the axis was only hundreds of millions so I did millions, is that alright?

(I'll add the png below!)

**Review by agitter — APPROVED** [review]
The updated image looks much cleaner.  Let's resolve the stats conflicts, merge, and see how the new variables work.

**Pull request merged**